cpp/build.sh

```diff
@@ -350,10 +342,12 @@ trap 'cd $BASE_DIR' EXIT
 cd $BUILD_DIR || exit
```

Comment:
Why do we jump into BUILD_DIR here and use pwd to get BWD? could we not directly use BUILD_DIR instead of BWD?